draft-privacypass-rate-limit-tokens.md

```diff
@@ -1166,6 +1167,7 @@ In pseudocode, this is as follows:
 
 ~~~
 sk_blind = BKS-BlindKeyGen()
+ctx = concat(encode(2, token_type)), "ClientBlind")
```

Comment:
Is there something else we want to bind in the context?

Not thinking of anything in particular but just the token type and a string does not seem like it's binding a lot.

Maybe something to take another look at in the pats-analysis?

Reply:
So I think we need to this context to say "this is client blinding for the rate limited privacy pass protocol," and I think what we have suffices for that. But I'm open to adding more if we think it's needed.